Project-Aurora/Project-Aurora/Devices/Ducky/DuckyRGBAPI.cs

```diff
@@ -0,0 +1,618 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Aurora.Devices.Ducky
+{
+    class DuckyRGBAPI
+    {
+        private static Dictionary<string, byte[][]> controlCommands = new Dictionary<string, byte[][]>
+        {
+            {"Shine_7_Takeover", new byte[][]
```

Comment:
Storing this as a static object instead of a value in a dictionary makes more sense to me. that way you'd be able to do 
```cs
foreach(var packet in DuckyRGBAPI.Takeover)
{
//do whatever
}
```
instead of having random strings everywhere.
IMO the class name doesnt make sense. you either store all ducke methods here and call them from the device integration, or rename this something like `DuckyConstants`.